fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -257,6 +257,14 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int label_keep_max_second = 3 * 24 * 3600; // 3 days
 
+    /**
+     * Transaction between [skip_transaction_range[0], skip_transaction_range[1]] will be removed,
+     * even though they are not in final status.
+     * Used for temporary solve issues
+     */
+    @ConfField(mutable = true)
+    public static long[] skip_transaction_range = {};
```

Comment:
I don't think it's suitable to merge it into main branch, we still need to find the root cause and avoid the same issue in the future.